src/test/kotlin/com/msd/application/GameMapServiceTest.kt

```diff
@@ -21,20 +22,16 @@ class GameMapServiceTest(
     @Autowired val gameMapService: GameMapService
 ) {
 
-    companion object {
-        val mockGameServiceWebClient = MockWebServer()
+    val mockGameServiceWebClient = MockWebServer()
 
-        @BeforeAll
-        @JvmStatic
-        internal fun setUp() {
-            mockGameServiceWebClient.start(port = 8081)
-        }
+    @BeforeEach
```

Comment:
Why did you change this to a `BeforeEach`? Doesn't it now take longer to run those tests?

Reply:
This somehow fixed the mockServer stopping to work after a few test methods